### PR TITLE
Add help on repository

### PR DESCRIPTION
**Pull request by yroffin**
Add an help content on repository with a default key on lhasa

capture d ecran 2018-07-03 a 11 13 48

### CODE REVIEW

File: webui/src/app/components/appdetail/appdetail.component.html [hunk 1]
```diff
@@ -35,7 +35,7 @@ <h5 class="oui-header_5">
       <div>
         <hr>
         <h5 class="oui-header_5">
-          <i class="oui-icon oui-icon-page_line" aria-hidden="true"></i>
+          <app-help-widget key="help-what-is-a-repository"></app-help-widget><i class="oui-icon oui-icon-page_line" aria-hidden="true"></i>
           {{ 'REPOSITORY' | translate }}
         </h5>
```
Comment (rayene) [inline webui/src/app/components/appdetail/appdetail.component.html]: The icon should appear *after* the string `Repository`.

File: samples/mycompany.sh [hunk 2]
```diff
@@ -27,6 +27,15 @@ An application is a piece of software that
...
+  --header 'content-type: text/plain' \
+  --data 'Code repository
+------
+
+Code of application is stored in a __code repository__ (often git)
+'
+
 curl --request PUT \
   --url $API_BASE_URL/v1/environments/prodfr \
   --header 'content-type: application/json' \
```
Comment (rayene) [inline samples/mycompany.sh]: Should we put default values in the DB migration scripts ?

Comment (yroffin) [reply samples/mycompany.sh]: yes on integration side

File: webui/src/app/components/appdetail/appdetail.component.html (hunk 1 above)
Comment (yroffin) [reply webui/src/app/components/appdetail/appdetail.component.html]: Done

**Pull request merged**